lib/tests/subscribe.rb

```diff
@@ -163,28 +167,23 @@ def validate_notify xml_s, schema_file="schemas/reginfo.xsd"
   t.add_quaff_scenario do
     call = ep1.outgoing_call(ep1.uri)
 
-    call.send_request("REGISTER", "", { "Expires" => "3600", "Authorization" => %Q!Digest username="#{ep1.private_id}"! })
+    call.send_request("REGISTER", "", { "Expires" => "3", "Authorization" => %Q!Digest username="#{ep1.private_id}"! })
     response_data = call.recv_response("401")
     auth_hdr = Quaff::Auth.gen_auth_header response_data.header("WWW-Authenticate"), ep1.private_id, ep1.password, "REGISTER", ep1.uri
     call.update_branch
 
-    # If testing against something with a min-expires set for registers, alter the 'Expires'
-    # value below to be just above it, and the sleep value below this to be just less than it
-    # i.e. if min-expires is set to 300, set Expires => 301, and sleep 295
-    #
-    # If the min-expires value is longer than 60 seconds, also change t.join(60) in lib/test-definition
-    # to be longer than the expires header in the REGISTER sent here.
     call.send_request("REGISTER", "", {"Authorization" => auth_hdr, "Expires" => "3"})
     response_data = call.recv_response("200")
 
     sub = ep2.outgoing_call(ep1.uri)
     sub.send_request("SUBSCRIBE", "", {"Event" => "reg"})
-    # 200 and NOTIFY can come in any order, so expect either of them, twice
+
+    # 200 and NOTIFY can come in any order, so expect either of them in response
```

Comment:
Can you add comments explaining what this NOTIFYs is?

We should probably be verifying the contents of the NOTIFY as well..